generate-docs.js

```diff
@@ -50,14 +50,14 @@ function replaceSearchbar(path) {
 
 function replaceUnderlinePrefixAndHtmlSuffixFromLinks(path){
   const html = fs.readFileSync(path, { encoding: 'utf8' });
-  const replaced = html.replace(/<a href="[^>]*_[^>]*.html">/gi, removeUnderlinePrefixAndHtmlSuffixFromLink);
+  const replaced = html.replace(/<a href="[^>]*_[^>]*.html[^>]*>/gi, removeUnderlinePrefixAndHtmlSuffixFromLink);
   fs.writeFileSync(path, replaced, { encoding: 'utf8' });
 }
 
 function removeUnderlinePrefixAndHtmlSuffixFromLink(aHref) {
   const i = aHref.indexOf('_');
-  // remove the first `_` and `.html`
-  return aHref.substring(0, i) +  aHref.substring( i+1, aHref.length - 7 ) + '\">'
+  const firstUnderlineRemoved = aHref.substring(0, i) +  aHref.substring( i + 1);
```

Comment:
Maybe leave the comment, I think it was helpful, also:
```suggestion
  const firstUnderlineRemoved = aHref.substring(0, i) +  aHref.substring(i + 1);
```